nixos/modules/installer/cd-dvd/sd-image-raspberrypi4.nix

```diff
@@ -20,12 +20,23 @@
     firmwareSize = 128;
     # This is a hack to avoid replicating config.txt from boot.loader.raspberryPi
     populateFirmwareCommands =
-      "${config.system.build.installBootLoader} ${config.system.build.toplevel} -d ./firmware";
+      "${config.system.build.installBootLoader} ${config.system.build.toplevel} -d ./";
```

Comment:
Why is this line even necessary?

As far as I can tell, it only runs for the initial SD card image build... and yet, my bootloader is properly configured everytime that I update the system, so I don't understand what this is doing one-time that is needed?

Reply:
Yep, very good point. I think with the removal of `./firmware`, it's probably not necessary at all.  
I'll remove it and take it for a spin later when I get home